templates/deployments.yaml

```diff
@@ -43,6 +43,24 @@ spec:
       {{- end }}
       # Sidecar container for generating .htpasswd with st2 username & password pair and sharing produced file with the main st2auth container
```

Comment:
These comment needs to be moved closer to the initContainer with beforementioned htpasswd

Reply:
Done